### DISCUSSION TITLE
Geforce GTX 1650 Ti and Vulkan

### DISCUSSION THREAD

**Discussion by JZeravik** [upscayl/upscayl#492] [Q&A] [answered]
HI guys, 

I have Geforce GTX 1650 Ti  it seem this card should support Vulcan whatever it is, I tried reinstall drivers but upscaler doesn't work, I have also set GPUID to 0, but it didn't help,  where could be problem?

**Comment by NayamAmarshe** [comment]
Yes, that happens sometimes. Reinstalling DirectX, VCRedist can sometimes fix this issue. Try the DirectX Repair: https://blog.csdn.net/VBcom/article/details/6962388

**Reply by JZeravik** [reply, marked as answer]
Thanks didn't try, (I 'm suspicious to these chinese installers) also found this and it really works: https://nvidia.custhelp.com/app/answers/detail/a_id/5182/~/unable-to-launch-vulkan-apps%2Fgame-on-notebooks-with-amd-radeon-igpus

**Reply by NayamAmarshe** [reply]
Sweet! Glad to know. We'll add this link to the wiki 😄
